docs/image_management/upload_images.md

```diff
@@ -0,0 +1,82 @@
+---
+description: Upload images to Ibexa DXP to manage them.
+---
+
+# Upload and store images
+
+!!! note
+    Some options may not be available depending on your configuration and permissions.
+
+## Manage folders
+
+By default, uploaded files are stored in the Media root folder.
+If you want to upload them to different location, navigate to the folder you want to upload to.
+
+### Add folders
+
+1\. Go to **Content** -> **Media**.
+
+2\. In the upper-right corner click **Create content**.
+
+3\. From the available Content types, select Folder and click Create.
+
+4\. Enter name for the new folder.
+
+5\. (Optional) Add folder description.
+
+6\. Click **Publish**.
+
+The newly created folder is now visible in the Content Tree structure.
+You can create subfolders by clicking the folder and repeating the above procedure.
+
+## Add image
+
+1\. Go to **Content** -> **Media**.
+
+2\. In the upper-right corner click **Create content**.
+
+3\. From the available Content types, select **Image** and click **Create**.
+
+4\. Provide image title.
+
+5\. In the **Asset** section, click **Upload file** and from the system browser, select an image.
+
+6\. To find images using tags, in the **Tags** field, enter keywords.
+
+Tagging image assets helps keep the media library organized and enables searching images by tags in the DAM.
+
+7\. Click **Publish**.
+
+!!!note 
+    The maximum file size and file extension limitations may differ depending on your configuration.
```

Comment:
Have we described how to limit image file extensions? 

![Screenshot 2024-01-09 at 09 47 44](https://github.com/ibexa/documentation-user/assets/9850711/bc3fa499-44b2-408c-8b08-93220fd88824)